INSTALL.md

```diff
@@ -115,11 +123,11 @@ $ sudo mysql -u root -p
 Example mysql configuration in *.env* file:
```

Comment:
@richarddushime here we specify the content goes in the `.env` file (to be parsed by `django-dotenv`)